code/game/mecha/mecha_control_console.dm

```diff
@@ -60,12 +60,9 @@
 	if(href_list["lockdown"])
 		var/obj/item/mecha_parts/mecha_tracking/MT = topic_filter.getObj("lockdown")
 		var/obj/mecha/M = MT.in_mecha()
-		if(M.state)
-			to_chat(usr, "That exosuit is already under lockdown!")
-			return
 		switch(alert("Are you sure? The exosuit will enter maintenance mode.","Transmit Beacon Lockdown Code","Yes","No"))
 			if ("Yes")
-				MT.lockdown()
+				to_chat(usr, "<span class = 'notice'>You [MT.lockdown()?"lock down":"remove the lockdown"] on \the [M]</span>")
 				log_game("[key_name_admin(usr)] locked down [M] via exosuit control console.")
```

Comment:
Probably want to add the same tenary message to the log message so it doesn't look like two lockdowns, but rather a lockdown and removed lockdown.

Reply:
good spot